packages/core/src/__tests__/ceramic-api.test.ts

```diff
@@ -467,6 +439,7 @@ describe('Ceramic API', () => {
     const notExistDocId = DocID.fromString('kjzl6cwe1jw1495fyn7770ujykvl1f8sskbzsevlux062ajragz9hp3akdqbmdg')
 
     beforeAll(async () => {
+      ceramic = await createCeramic()
```

Comment:
ugh.  So I'm guessing these tests worked if they were run after the previous tests, but failed if run on their own because the `ceramic` object hadn't been set?  In any case if we're adding this to `beforeAll` we probably need a corresponding `afterAll` to close it right?

Reply:
Yes! Sad thing about it I has just wiped the changes that make it concur with your logic. Restoring.

Reply:
And, restored.